[2021-11-29 12:29:11,023] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-01-28T00:00:00+00:00 [queued]>
[2021-11-29 12:29:11,077] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-01-28T00:00:00+00:00 [queued]>
[2021-11-29 12:29:11,081] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:29:11,082] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-11-29 12:29:11,083] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:29:11,128] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_1> on 2020-01-28T00:00:00+00:00
[2021-11-29 12:29:11,140] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2020-01-28T00:00:00+00:00', '--job-id', '187', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpa2i5cglb', '--error-file', '/tmp/tmpfep1mwpy']
[2021-11-29 12:29:11,144] {standard_task_runner.py:52} INFO - Started process 135347 to run task
[2021-11-29 12:29:11,156] {standard_task_runner.py:77} INFO - Job 187: Subtask task_1
[2021-11-29 12:29:11,300] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2020-01-28T00:00:00+00:00 [running]> on host airflowvm
[2021-11-29 12:29:11,438] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2020-01-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-28T00:00:00+00:00
[2021-11-29 12:29:11,439] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-11-29 12:29:11,440] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-11-29 12:29:11,451] {subprocess.py:75} INFO - Output:
[2021-11-29 12:29:14,462] {subprocess.py:83} INFO - Command exited with return code 0
[2021-11-29 12:29:14,499] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20200128T000000, start_date=20211129T122911, end_date=20211129T122914
[2021-11-29 12:29:14,538] {taskinstance.py:1245} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-11-29 12:29:14,567] {local_task_job.py:151} INFO - Task exited with return code 0
